.github/workflows/push-tag.yml

```diff
@@ -0,0 +1,88 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+name: '[CI/CD] Push tag'
+on: # rebuild any PRs and main branch changes
+  push:
+    branches:
+      - main
+    paths:
+      - 'bitnami/**'
+      - '!**.md'
+# Remove all permissions by default.
+permissions: {}
+jobs:
+  get-chart:
+    runs-on: ubuntu-latest
+    name: 'Get modified charts'
+    permissions:
+      contents: read
+    outputs:
+      chart: ${{ steps.get-chart.outputs.chart }}
+      result: ${{ steps.get-chart.outputs.result }}
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+        with:
+          path: charts
+          fetch-depth: 2 # to be able to obtain files changed in the latest commit
+      - id: get-chart
+        name: 'Get modified charts'
+        run: |
+          cd charts
+          files_changed="$(git show --pretty="" --name-only)"
+          # Adding || true to avoid "Process exited with code 1" errors
+          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "bitnami/[^/]*" | sort | uniq || true)"
```

Comment:
```suggestion
          charts_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "bitnami/[^/]*" | sort -u || true)"
````

Reply:
I do not think we should address that kind of topic here.
That section remains unchanged from the original workflow.

In this PR these are the changes I performed:

- Renamed the file
- Removed the 'vib-publish' step
- Removed the 'update-index' step
- Removed the 'push-promotion' step
- Removed the 'slack-notif' step

Because of the file rename changes may not be as visible as they should, I will try to address that.

Reply:
It should be easier to see now that I separated it in two commits: https://github.com/bitnami/charts/pull/28178/commits/0433d24ae556a9ac2d9cff4f45a1c69096e1a542

Although Github Diff still shows the changes as removing an entire file and adding it again.